pkgs/development/libraries/libadwaita/default.nix

```diff
@@ -2,35 +2,40 @@
 , stdenv
 , fetchFromGitLab
 , docbook-xsl-nons
+, gi-docgen
 , gtk-doc
+, libxml2
 , meson
 , ninja
 , pkg-config
 , sassc
 , vala
 , gobject-introspection
+, fribidi
 , gtk4
 , xvfb-run
 }:
 
 stdenv.mkDerivation rec {
   pname = "libadwaita";
-  version = "unstable-2021-05-01";
+  version = "1.0.0-alpha.1";
 
-  outputs = [ "out" "dev" "devdoc" ];
-  outputBin = "dev";
+  outputs = [ "out" "dev" "doc" ];
+  outputBin = "doc";
```

Comment:
Does this make sense given that the only binary is a demo application?

Reply:
Makes sense. Though I wonder if we should not still save it to `devdoc`.

Reply:
I changed devdoc to dev cause after the update docs land in `share/doc` instead of `share/gtk-doc`.

Reply:
Yeah, gi-docgen now puts them there so the our multiple-outputs setup hook will move them to doc output instead. But this is still a API docs so :question:

Reply:
So you suggest I add `mv $out/share/doc $out/share/gtk-doc` to `preFixup`?

Reply:
Either that or `mv $out/share/doc $doc/share/doc` in `postInstall`. Though, your proposal will also help with Devhelp (it will only start looking to `share/doc` in the next release).